lib/ex_typesense/collection.ex

```diff
@@ -294,7 +294,8 @@ defmodule ExTypesense.Collection do
   @doc """
   Permanently drops a collection by collection name or module name.
 
-  **Note**: dropping a collection does not remove the referenced alias, only the indexed documents.
+  **Note**: dropping a collection does not remove the referenced
+  alias, only the indexed documents.
```

Comment:
**question:** Is this really correct? We drop the collection altogether right, not just the indexed documents?

Reply:
@shijithkjayan here's the details:

- dropping collection will also delete all documents
- dropping collection will not affect alias
- removing all documents will not drop the collection

see `test/collection_test.exs` for the updates. LMK your thoughts on this.

Reply:
@shijithkjayan the reason why alias is not removed when dropping collection is because of this example scenario:

A new collection (including its documents) was created and it replaces the dropped collection and points it to the existing alias.

>  - This is especially helpful when you need to update the schema of a collection. You'd create a new timestamped collection, re-index your data in it and then swap the alias to point to it. [source](https://typesense.org/docs/guide/running-in-production.html#schema-management)
>  - You want to use the [Collection Alias](https://typesense.org/docs/26.0/api/collection-alias.html) feature to do a reindex without any downtime into a new collection and then switch the alias over to the new collection. [source](https://typesense.org/docs/26.0/api/collection-alias.html#collection-alias)